RecoDataProducts/inc/TrkFitFlag.hh

```diff
@@ -18,7 +18,7 @@ namespace mu2e {
     enum bit_type {hitsOK=0,circleOK,phizOK,helixOK,seedOK,kalmanOK,circleInit,phizInit,
     circleConverged,phizConverged,helixConverged,seedConverged,kalmanConverged,
     MatCorr, BFCorr, FitOK,
-    KSF=16, KFF, TPRHelix, CPRHelix, Straight, KKLoopHelix,KKCentralHelix,KKLine};
+    KSF=16, KFF, TPRHelix, CPRHelix, Straight, KKLoopHelix,KKCentralHelix,KKLine, MCSeed=31};
```

Comment:
Yes, thanks for catching this, I'll make it consistent.  MCSeed is incomplete development in this branch: the structure is in place but the implementation is incomplete.  It will be addressed in a future PR.